aspnet/fundamentals/localization/sample/Controllers/AccountController.cs

```diff
@@ -0,0 +1,467 @@
+﻿using System;
```

Comment:
It doesn't seem like this Controller is referenced by the doc, can we trim it to reduce clutter?